recipe/migrations/osx_arm64.txt

```diff
@@ -215,6 +215,7 @@ cyanure
 cyipopt
 cython-fortran-file
 dagster
+daqp
```

Comment:
Title didn't match the feedstock being added here, but I checked that it's https://github.com/conda-forge/daqp-feedstock, not dqap

Reply:
Whoops, sorry, thanks for catching this!